wordpress/README.md

```diff
@@ -93,17 +93,17 @@ The database container name is [project name]-db.
 
 **PARAMETERS**
 
-While activating the WordPress development environment as single unit (main.mlab), the main container controls the parameters.
+While activating the WordPress development environment as a single unit (main.mlab), the main container controls the parameters.
 
-* Database IP Address. The IP address the database.
-* project name. The project name is used as a unique identifier of a project.
+* Database IP Address. The database's IP address.
+* project name. Used as the project unique identifier.
 
 **ACTIONS**
 
 * mysql - opens a mysql command-line client.
 * shell - open a bash command-line window connected to the container.
 
-### `phpMyAdmin.mlab` - an administration of MySQL / MariaDB over the Web
+### `phpMyAdmin.mlab` - MySQL administration / MariaDB over the Web
```

Comment:
The should probably be MySQL/MairaDB administration over he web

Reply:
Corrected and committed. do you see the correction (i'm not sure now the whole thing works in GitHub...)?